moped-editor/src/app/App.jsx

```diff
@@ -29,6 +44,7 @@ const App = () => {
         </MatxTheme>
       </Provider>
     </AppContext.Provider>
+     </ApolloProvider>
```

Comment:
These are fine, but it would be nice to have some stylistic tabulation to make it easier to read.